crates/rover-client/src/query/config/whoami.graphql

```diff
@@ -0,0 +1,10 @@
+query WhoAmIQuery {
+  me {
+    __typename
+    name
+    id
```

Comment:
While I think it's reasonable to display IDs for debugging, I'll note that we don't actually show the user ID to users on the Studio UI.  For first-party users, this is just `fp.UUID`, and as of recently new GitHub users are also `gh.UUID` (and for older GitHub users, the ID might not actually match their username if they've changed it on GitHub). So while it's probably reasonable to display the ID, it would be very valuable to also show `... on User { email githubUsername }`.